setup.py

```diff
@@ -2,8 +2,89 @@
 # -*- encoding: utf-8 -*-
 
 from setuptools import setup
+from distutils.command import build
+from distutils.command import install
+#from distutils.command import install_scripts
+import distutils
+import subprocess
+
 from kiwi.version import __version__
 
+
+class kiwi_build(build.build):
```

Comment:
Ahh ok. I've tried it, but it didn't give me invite loops. :) Worked for me. :) But it's only a minor thing.